From 58cd5e3f174d29bfb4dd9ba89a1d614b39221bd1 Mon Sep 17 00:00:00 2001
From: smeikx <pack@man.vim>
Date: Fri, 6 Mar 2020 22:02:54 +0100
Subject: [PATCH] prevent overwriting indentation settings

---
 ftplugin/graphql.vim | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/ftplugin/graphql.vim b/ftplugin/graphql.vim
index 55cf447..bafb4e8 100644
--- a/ftplugin/graphql.vim
+++ b/ftplugin/graphql.vim
@@ -30,8 +30,5 @@ setlocal comments=:#
 setlocal commentstring=#\ %s
 setlocal formatoptions-=t
 setlocal iskeyword+=$,@-@
-setlocal softtabstop=2
-setlocal shiftwidth=2
-setlocal expandtab
 
 let b:undo_ftplugin = 'setlocal com< cms< fo< isk< sts< sw< et<'
-- 
2.25.0

